### ISSUE TITLE
Empty generated waves

### ISSUE THREAD

**Issue by hmubarak** [Kyubyong/tacotron#31]
Hi,
Thank you so much for your great work!

To save training time, I took the wave files under Genesis folder, and updated the text.csv file accordingly.
Train was done for these 1532 files successfully as shown below:

**python train.py**
Training Graph loaded
2017-06-08 10:35:51.200597: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2017-06-08 10:35:51.200628: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-06-08 10:35:51.200633: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
Done

Then, I ran the evaluation script which generated a text file with 31 sentences and their waves.
**python eval.py**
Graph loaded
2017-06-08 13:22:30.452388: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2017-06-08 13:22:30.452540: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-06-08 13:22:30.452657: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
WARNING:tensorflow:Standard services need a 'logdir' passed to the SessionManager
Restored!
Done

**The problem is that all the generated wave files are almost empty with some noise in the middle until the end.**

This is one of the these generated files: 
(model_epoch_200_gs_1000_0.zip)

Any idea how to fix this issue?
Many thanks,
Hamdy

**Comment by Kyubyong** [comment]
Can you share your training loss graph?

**Comment by hmubarak** [comment]
Thanks for you fast reply!
Could you please tell me where I can find this file?

**Comment by Kyubyong** [comment]
From the TensorBoard. I guess 200 epochs, or 1000 global steps are far from enough. Note that the paper said they trained for 2 million global steps. And I think that's one of the biggest reasons why people including me have had difficulties getting a good result. (The other reason is, of course, some possible bugs in the implementation. But it's also true that mostly we don't have enough gpus or time or patience!) But the training curve keeps slowly decreasing, I think that's a good sign.

**Comment by candlewill** [comment]
@hmubarak I have the same problem here with yours. 

All of the generated voice is like this: 
![image](https://user-images.githubusercontent.com/4916563/26957675-2ecf871e-4cf9-11e7-884d-9fae63b313ff.png)

Loss
![image](https://user-images.githubusercontent.com/4916563/26957712-706fdb4c-4cf9-11e7-944f-28c412e82282.png)

**Comment by Spotlight0xff** [comment]
@candlewill As said before, we need to be more patient with the training. 800 global steps is very unlikely to yield good results.

**Comment by frozen-finger** [comment]
I faced the same problem. Did you solve this problem by keeping training?(i am now over 26k steps)

**Comment by giridhar-pamisetty** [comment]
@candlewill @Kyubyong @hmubarak How did you solve this issue? I trained for almost 50k steps. But still when I synthesize using synthesize.py, it is generating empty wavs. Can you please solve this issue?
